src/Splash.tsx

```diff
@@ -74,8 +74,8 @@ export function Splash(props: React.PropsWithChildren<Props>) {
     isLayoutReady &&
     reduceMotion !== undefined
 
-  const colorScheme = useColorScheme()
-  const isDarkMode = colorScheme === 'dark'
+  const colorTheme = useColorModeTheme()
+  const isLightMode = colorTheme === 'light'
```

Comment:
Flip the condition because we want to match both `'dark'` and `'dim'`. So do it by excluding `'light'` and flipping the cases below.